packages/marketplace/src/components/ui/unsupport-brower-pop-up.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { ToastMessage } from '@reapit/elements'
+import styles from '@/styles/blocks/sandbox-pop-up.scss?mod'
+import { useSelector } from 'react-redux'
+import { selectRefreshSession } from '@/selector/auth'
+
+export const TIMEOUT_DURATION = 500
+
+export const popUp = (setOpen, unsupported) => () => {
+  if (unsupported) {
+    const timeout = setTimeout(setOpen, TIMEOUT_DURATION)
+    return () => {
+      clearTimeout(timeout)
+    }
+  }
+}
+
+export const UnsupportBrowerPopUp = ({
+  unsupported = false,
+  // eslint-disable-next-line
+  message = 'Unsupported Browser - Unfortunately as Internet Explorer 11 is no longer supported, we are unable to display the documentation, please login to the Developer Portal using either Chrome, Firefox, Safari or Edge',
```

Comment:
It isn't always developer portal, it can be client or admin portal too, so the message here doesn't make sense